design-system/scripts/generate-design-tokens.js

```diff
@@ -68,45 +152,53 @@ Object.entries(definitions.decisionGroupsByTheme).forEach(
         }
         if (
           !designTokens[themeName][decision.choice] &&
-          !designTokens.default[decision.choice]
+          !designTokens.default[decision.choice] &&
+          !isAllowedCssChoice(decision.choice)
```

Comment:
This is to support some CSS values in the design tokens choices.
We covering cases like this: secondary buttons currently don't use borders values but they do in the new themes. I added a new token to handle the difference and I'm using the value `none` for the default theme but `1px solid hsla(0, 0%, 81%, .60)` in the new theme.

Also, height values don't use a scale because they are very coupled to the component needing them so I'm using pixel values: `height-for-button-as-big: 32px`